### PR TITLE
Support 1.20.6

### PR DESCRIPTION
**Pull request by BoomEaro**
PacketRegistryData at the configuration stage was changed in 1.20.5 and now requires sending each packet separately with each registry type.

### CODE REVIEW

File: src/main/java/ua/nanit/limbo/protocol/registry/State.java [hunk 1]
```diff
@@ -83,36 +83,44 @@ public enum State {
...
                     PacketFinishConfiguration::new,
-                    map(0x02, V1_20_2, V1_20_3)
+                    map(0x02, V1_20_2, V1_20_3),
+                    map(0x03, V1_20_5, V1_20_5)
             );
             clientBound.register(
                     PacketKeepAlive::new,
-                    map(0x03, V1_20_2, V1_20_3)
+                    map(0x03, V1_20_2, V1_20_3),
+                    map(0x05, V1_20_5, V1_20_5)
```
Comment (Cyramek) [inline src/main/java/ua/nanit/limbo/protocol/registry/State.java:102]: https://github.com/PaperMC/Velocity/blob/dev/3.0.0/proxy/src/main/java/com/velocitypowered/proxy/protocol/StateRegistry.java#L188

Comment (BoomEaro) [reply src/main/java/ua/nanit/limbo/protocol/registry/State.java]: Thanks!

**Comment by lucian929** [comment]
Thanks for this PR!

**Comment by carlodrift** [comment]
Tested and works perfectly! Great job.

**Comment by tavaresjoshua8** [comment]
README should also change to show compatibility with 1.20.5
image

**Comment by Nan1t** [comment]
Good job 👍, thank you! I will accept some other changes and release it soon.

**Pull request merged**